parlai/tasks/coco_caption/build_2017.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+# Download and build the data if it does not exist.
+
+import parlai.core.build_data as build_data
+import os
+
+
+def buildImage(opt):
+    dpath = os.path.join(opt['datapath'], 'COCO-IMG')
```

Comment:
(As Alex said) we should make this be a different directory, e.g. `COCO-IMG-2017` - otherwise, if we have already built `COCO-IMG` but don't have the 2017 images, this build function will remove all the downloaded images (see line 18).